Sources/ContentsquareRemoteCommand.swift

```diff
@@ -85,4 +91,36 @@ public class ContentsquareRemoteCommand: RemoteCommand {
             }
         }
     }
+    
+    func customVarsFromArrays(_ payload: [String: Any]) -> [[String: Any]] {
+        let customVarArrays = payload.normalizeCustomVarArrays()
+        // We assume that all arrays have the same length
+        let count = customVarArrays.first?.value.count ?? 0
+        var result = Array<[String: Any]>(repeating: [:], count: count)
+        
+        for i in 0 ..< count {
+            if let indexes = customVarArrays[ContentsquareConstants.CustomVars.indexes], indexes.count > i {
+                result[i]["index"] = indexes[i]
+            }
+            if let names = customVarArrays[ContentsquareConstants.CustomVars.names], names.count > i {
+                result[i]["name"] = names[i]
+            }
+            if let values = customVarArrays[ContentsquareConstants.CustomVars.values], values.count > i {
+                result[i]["value"] = values[i]
+            }
+        }
```

Comment:
The reason why we filtered the keys in `normalizeCustomVarArrays` is precisely to avoid repeating these lines for all keys. 
Once the keys are unified and put in singular, we can just write this:
```suggestion
        for i in 0 ..< count {
            for key in customVarArrays.keys {
                if let values = customVarArrays[key], values.count > i {
                    result[i][key] = values[i]
                }
            }
        }
```